packages/running/style/base.css

```diff
@@ -181,29 +177,6 @@ span.jp-RunningSessions-icon > svg {
 
 .jp-RunningSessions-shutdownAll.jp-ToolbarButtonComponent {
   color: var(--jp-warn-color1);
-  background-color: transparent;
-  border-radius: 2px;
-  overflow: hidden;
-  white-space: nowrap;
-  text-overflow: ellipsis;
-}
-
-.jp-RunningSessions-shutdownAll.jp-ToolbarButtonComponent:hover {
-  background-color: var(--jp-layout-color2);
-}
-
-.jp-RunningSessions-shutdownAll.jp-ToolbarButtonComponent:focus {
-  border: none;
-  box-shadow: none;
-  background-color: var(--jp-layout-color2);
-}
-
-.jp-RunningSessions-shutdownAll.jp-ToolbarButtonComponent[disabled] {
-  color: var(--jp-ui-font-color2);
-}
-
-.jp-RunningSessions-shutdownAll.jp-ToolbarButtonComponent[disabled]:hover {
-  background: none;
```

Comment:
Is this intentional? I think that previous version with grey color and without hover background for disabled button was better

![image](https://github.com/jupyterlab/jupyterlab/assets/5832902/0a7b9303-97b9-49d0-baab-98adf97822ca)